app/graphql/albums/resolvers.js

```diff
@@ -5,20 +5,26 @@ exports.getAlbum = async (_, params) => {
   const data = {
     ...album,
     artist: album.userId,
-    name: album.title,
-    photos: await photosByAlbumId(params.id)
+    name: album.title
```

Comment:
Album shouldn't have field `name`

Reply:
will be corrected in https://github.com/wolox-training/jb-graphql/pull/7#discussion_r313487397